core/oio_sds.h

```diff
@@ -330,6 +337,27 @@ struct oio_error_s* oio_sds_delete (struct oio_sds_s *sds, struct oio_url_s *u);
 struct oio_error_s* oio_sds_has (struct oio_sds_s *sds, struct oio_url_s *url,
 		int *phas);
 
+
+typedef void (*On_Element) (void *ctx, char *key, char *value);
+	
+/* Get properties of a file : v will be allocated */
+struct oio_error_s* oio_sds_get_content_properties (struct oio_sds_s *sds,
+						    struct oio_url_s *url,
+						    On_Element fct, void* ctx);
```

Comment:
Please don't mix camel case and snake case (we prefer snake case). And indent with 2 tabs after breaking a line.